src/components/Landing/components/ContactUs/ContactUs.tsx

```diff
@@ -0,0 +1,36 @@
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+export default function ContactUs() {
+  const { t } = useTranslation();
+
+  return (
+    <div className="section contactus-wrapper">
+      <div className="section-content">
+        <div className="contactus-form">
+          <p className="contactus-form__title">
+            {t("Landing.ContactUs.title")}
+          </p>
+          <p className="contactus-form__subtitle">
+            {t("Landing.ContactUs.subtitle")}
+          </p>
+          <div className="contactus-form__element">
+            <label>{t("Landing.ContactUs.name")}</label>
+            <input type="text" />
+          </div>
+          <div className="contactus-form__element">
+            <label>{t("Landing.ContactUs.email")}</label>
+            <input type="text" />
+          </div>
+          <div className="contactus-form__element">
+            <label>{t("Landing.ContactUs.position")}</label>
+            <input type="text" />
+          </div>
+          <button className="button contactus-form__button">
```

Comment:
@mwogates Let's just make the button disabled when at least one of the fields is empty.
I would also add validation for the email address with a red error in case it's invalid email. The send button should be disabled in that case too.